### PR TITLE
Include country in simplified

### PR DESCRIPTION
**Pull request by ithielnor**
Since the output is grouped by country, it would be nice to include this in the output.

### CODE REVIEW

**Comment by vvo** [comment]
Thanks! Being curious: how do you plan on using the country code? I was thinking of inlining it in the formatted output at some point.

**Comment by ithielnor** [comment]
I'm using it in my own formatted string, which I'd like to show as `(GMT-offset) countryname - cities (alt name)`

**Comment by vvo** [comment]
So I guess, no more showing the "alternative name", ok understood!

File: generate.js [hunk 1]
```diff
@@ -260,6 +261,7 @@ async function run() {
         ({
           name,
           alternativeName,
+          country,
```
Comment (vvo) [inline generate.js:264]:
```suggestion
          countryName,
```

File: generate.js [hunk 2]
```diff
@@ -268,6 +270,7 @@ async function run() {
           return {
             name,
             alternativeName,
+            country,
```
Comment (vvo) [inline generate.js:273]:
```suggestion
            countryName,
```

**Comment by ithielnor** [comment]
I'm including the alt name at the end in parenthesis.

File: generate.js [hunk 3]
```diff
@@ -230,6 +230,7 @@ async function run() {
         name: timeZoneName,
         alternativeName: alternativeTimeZoneName,
         group,
+        country: countries[countryCode],
```
Comment (vvo) [inline generate.js:233]:
```suggestion
        countryName: countries[countryCode],
```

**Pull request merged**